charts/kyverno/values.yaml

```
@@ -14,118 +14,118 @@ kyverno:
-            - create
-            - delete
-            - get
-            - list
-            - patch
-            - update
-            - watch
-          - apiGroups:
-            - metallb.io
-            resources:
-            - L2Advertisement
-            verbs:
-            - get
-            - list
-            - patch
-            - update
-            - watch
-          - apiGroups:
-            - postgresql.cnpg.io
-            resources:
-            - backups
-            verbs:
-            - get
-            - list
-            - patch
-            - update
-            - watch
+        - apiGroups:
+          - networking.k8s.io
+          resources:
+          - ingresses
+          - ingressclasses
+          - networkpolicies
+          verbs:
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - rolebindings
+          - roles
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - rbac.authorization.k8s.io
+          resources:
+          - clusterroles
+          verbs:
+          - bind
+        - apiGroups:
+          - ''
+          resources:
+          - configmaps
+          - resourcequotas
+          - limitranges
+          verbs:
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          - project.openshift.io
+          resources:
+          - projectrequests
+          - projects
+          verbs:
+          - create
+          - get
+        - apiGroups:
+          - user.openshift.io
+          resources:
+          - groups
+          verbs:
+          - list
+          - get
+        - apiGroups:
+          - ''
+          resources:
+          - secrets
+          verbs:
+          - get
+          - list
+          - create
+          - update
+          - delete
+          - watch
+        - apiGroups:
+          - 'objectbucket.io'
+          resources:
+          - ObjectBucketClaim
+          verbs:
+          - get
+          - list
+          - watch
+        - apiGroups:
+          - k8s.ovn.org
+          resources:
+          - egressips
+          verbs:
+          - create
+          - delete
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - metallb.io
+          resources:
+          - L2Advertisement
+          verbs:
+          - get
+          - list
+          - patch
+          - update
+          - watch
+        - apiGroups:
+          - postgresql.cnpg.io
+          resources:
+          - backups
+          verbs:
+          - get
+          - list
+          - patch
+          - update
+          - watch
 
     resources:
       limits:
-        memory: 500Mi
+        memory: 800Mi
```

Let's increase the memory limit even more.. the memory should be the less problematic resource out of the CPU / RAM.
```suggestion
        memory: 1Gi
```